### PR TITLE
3.7 - remove phpstan/phpstan-phpunit 

### PR DESCRIPTION
**Pull request by saeideng**

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -32,6 +29,7 @@ parameters:
         - '#Call to an undefined method Traversable::getInnerIterator\(\)#'
         - '#PHPDoc tag @throws with type PHPUnit\\Exception is not subtype of Throwable#'
         - '#Binary operation "\+=" between \(array\&hasOffset\(.*\)\) and array results in an error#'
+        - '#Call to an undefined method PHPUnit\\Framework\\MockObject\\MockObjec#'
```
Comment (dereuromark) [inline phpstan.neon]: missing t

**Comment by dereuromark** [comment]
Is this really the best way forward here?

**Comment by saeideng** [comment]
>Is this really the best way forward here?
using of phpstan is hard in local , when  you want to use global phpstan ( PHAR archive) + with this dependency
it uses just in `Cake\TestSuite\TestCase::loadPlugins()`

**Comment by dereuromark** [comment]
That explanation would have been nice directly in the PR comment/description :)

**Comment by ADmad** [comment]
Yeah it's better to just ignore that error instead of adding an extra dependency.

**Comment by ADmad** [comment]
Errors reported by phpstan need to be fixed :)

**Pull request merged**